inference/core/cache/serializers.py

```diff
@@ -52,10 +52,19 @@ def build_condensed_response(responses):
         if not getattr(response, "predictions", None):
             continue
         try:
-            predictions = [
-                {"confidence": pred.confidence, "class": pred.class_name}
-                for pred in response.predictions
-            ]
+            predictions = []
+            for pred in response.predictions:
+                if type(pred) == str:
```

Comment:
please use `isinstance(pred, str)`

Reply:
if the clause is supposed to handle multi-label cls - then this is wrong approach whatsoever - please refer to response schemas